README.md

```diff
@@ -1,280 +1,261 @@
+clients. It allows importing a private key in order to act as a validator. In addition, it will start a
+[watchtower](https://hub.docker.com/r/containrrr/watchtower) to automatically update the Docker containers when newer versions
+become available (e.g. for bug fixes or network forks).
 
-On other operating systems you should check if your time is synchronized with an [NTP server](https://www.pool.ntp.org/).
+To fetch the most recent version of the quickstart script and run it, execute the following command on your machine:
 
----
+```sh
+$ wget -O quickstart.sh https://github.com/trustlines-protocol/blockchain/raw/master/quickstart.sh && bash quickstart.sh
+```
 
-### With Docker
+The script is interactive and will ask you which components to set up. Once the
+setup is complete, the various components will run in the background in the form
+of Docker containers. Configuration and chain data can be found in the
+`trustlines` directory placed in the current working directory. It is possible
+to customize the own setup by editing those configuration files. This goes for
+the configuration of the different components, as well as the composition of the
+Docker containers. If an optional component has not been setup on an earlier
+run, it can be added later by executing the quickstart script again.
 
-The following instructions explain how to start a local peer with the _Docker_ image. In fact it use a pre-configured [Parity
-Ethereum](https://www.parity.io/) client, combined with a set-up wrapper, to make connecting as easy as possible. The image is prepared to be
-used in three different scenarios: as observer, to participate and connect as validator.
+Executing the script again is safe: No configuration will be overridden without
+asking, in case the user has changed them itself. If conflicting configuration
+updates occur, they are shown to the user who can ask to see a diff of the
+changes.
 
-#### Pre-Requisites
+### Setup With Docker
 
-To make it work, a complete [Docker](https://www.docker.com/) environment is necessary to be installed on your system. Please take a look into
-the [official documentation](https://docs.docker.com/install/#general-availability) and use the instructions for your respective OS. Some
-niche operating systems are not mentioned there but still provide packages (e.g. [ArchLinux - docker](https://www.archlinux.org/packages/community/x86_64/docker/)).
-Make sure that your user is added to the `docker` user-group on _Unix_ systems, if you can not access root permissions to run containers.
+A more explicit way of setting up the various components is starting the Docker containers manually as described here.
+To keep commands as concise as possible, only the most basic options are provided. You might want to set additional
+ones, e.g., container names or restart policies. Notice that we intent to
+rebuild to setup of the quickstart script within the following sections. It is
```

Comment:
`Notice that we intent to rebuild to setup of the quickstart script within the following sections.`
I do not understand this sentence, if you are ok with removing it, please do.

Reply:
Copy of our _RocketChat_ conversation:
Hmm. My problem was that the descriptions by Jannik where quite “complicted” without a hint why they are like that. So he reconstructs the `trustlines` directory structure and all that. So in fact he is doing what the quickstart script is doing, but the manual way.